spec/ics-003-connection-semantics/README.md

```diff
@@ -290,12 +293,18 @@ function connOpenInit(
   desiredCounterpartyConnectionIdentifier: Identifier,
   counterpartyPrefix: CommitmentPrefix,
   clientIdentifier: Identifier,
-  counterpartyClientIdentifier: Identifier) {
+  counterpartyClientIdentifier: Identifier,
+  version: string) {
     abortTransactionUnless(validateConnectionIdentifier(identifier))
     abortTransactionUnless(provableStore.get(connectionPath(identifier)) == null)
     state = INIT
+    if version != "" {
+      versions = [version]
```

Comment:
should their be a check that the version supplied is contained in the compatible versions?

Reply:
It was manually provided, but I suppose, sure

Reply:
Added a check

Reply:
yea but what if the manually provided version isn't actually supported. Wouldn't the connection handshake be doomed to fail in connection ack or confirm?